website/app/components/doc/wcag-list/index.hbs

```diff
@@ -1,8 +1,8 @@
 <div class={{this.classNames}}>
-  <ul class="doc-wcag-list__list">
+  <ul class="doc-list-generic">
```

Comment:
we need to add the typographic styling (currently works because the element is wrapped in a `<p class="doc-markdown-p">`, but soon - once this https://hashicorp.atlassian.net/browse/HDS-1366 is done - will not work anymore)

```suggestion
  <ul class="doc-list-generic doc-text-body">
```